hcfb-be/set_config.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+echo "Connect to local database or production database? input: (local or production)"
+
+read databaseMode
+
+if [ $databaseMode = local ]
+then
+        echo Local Database Selected
+        echo Creating env.js file...
+        rm -f app/config/env.js
+        touch app/config/env.js
+        echo "const env = {
+                database: 'test',
+                username: 'postgres',
+                password: 'mysql',
+                host: 'localhost',
+                dialect: 'postgres',
+                pool: {
+                        max: 5,
+                        min: 0,
+                        acquire: 30000,
+                        idle: 10000
+                      }
+               };
+	       module.exports = env" >> app/config/env.js
+        echo "Local env.js config created, database: 'test', username: 'postgres', password: 'mysql'"
+	exit 0
+fi
+
+if [ $databaseMode = production ]
```

Comment:
```suggestion
if [ $databaseMode = "production" ]
```

Consistency, why not.

Reply:
Fixed